aten/src/ATen/native/native_functions.yaml

```diff
@@ -7804,6 +7804,12 @@
   device_check: NoCheck   # TensorIterator
   variants: method, function
 
+- func: argsort.stable(Tensor self, *, bool stable, int dim=-1, bool descending=False) -> Tensor
```

Comment:
Does the new overload have to be keyword-only in order to be able to distinguish between overloads? If not, then that's better left out, or perhaps applied to `stable` only.

Either way this looks a little confusing - why doesn't `stable` come at the end?

The docstring change which adds `*` seems incorrect either way, given that the existing signatures aren't keyword-only.

Reply:
- Stable with a default argument at the end is ambiguous --> (Tensor, int, bool, bool)
- Can't go there without a default argument (because arguments before it have defaults)
--> That's why it ended up in a non-standard location.
I was also influenced by `sort.dimname_values` which has a similar signature. It does not need to be a keyword only argument, but then the positions have been reordered.

Reply:
We were hitting some conflicts in the past, hence this specific order, https://github.com/pytorch/pytorch/pull/50052#issuecomment-766798770.